src/api/goods.ts

```diff
@@ -6,6 +6,12 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => {
+  return getAll()
+    .then(goods => goods.slice(0, 5));
+};
```

Comment:
missed [sorting](https://github.com/luckdenny/react_dynamic-list-of-goods#:~:text=Load%205%20first%20goods%20shows%205%20first%20goods%20after%20sorting%20them%20by%20name)